pkg/provider/testing/fake/fake.go

```diff
@@ -80,8 +81,8 @@ func (v *Client) GetAllSecrets(ctx context.Context, ref esv1beta1.ExternalSecret
 	return v.GetAllSecretsFn(ctx, ref)
 }
 
-// Not Implemented PushSecret.
 func (v *Client) PushSecret(_ context.Context, secret *corev1.Secret, data esv1beta1.PushSecretData) error {
+	fmt.Println("CALLING PUSH SECRET: ", secret.Data, data)
```

Comment:
I would use a proper logger instead of fmt to print the log or is this supposed to be just for debugging? 🤔 

- [Example](https://github.com/external-secrets/external-secrets/blob/3fbe318582266db5eefbbb11ad9ba1c1b3b795b0/pkg/provider/aws/secretsmanager/secretsmanager.go#L76)

Reply:
Oh snap. That's actually an accidentally left in debug. 🤣 thanks 🙏